public/js/sunnah.js

```diff
@@ -401,6 +401,7 @@
 		hadithStr += !grade		?	"" :  '\n' + 'Grade: ' + grade;
 		hadithStr += !hadithUrl	?	"" :  '\n' + hadithUrl;
 
+		console.log(hadithStr.trim());
```

Comment:
Do we still need the log statement here?

Reply:
This got passed on by mistake. However, I removed it at a later commit. So this mistake is not present in the latest code.